### PR TITLE
Removed files that seem to be unnecessary

### PR DESCRIPTION
**Pull request by dethos**
Please review from the tests I've made this files don't seem to be required.

### CODE REVIEW

**Comment by jlbribeiro** [comment]
@dethos: this seems good to go for me but I better have @MariCerv review this one, since she'll know the consequences to the codebase.

**Comment by pjaneiro** [comment]
@jlbribeiro Are we still waiting for @MariCerv's approval on this one?

**Review by pjaneiro — APPROVED** [review]
LGTM. Merging

**Pull request merged**
